src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+  const lessThanThreeDaysCost = dailyCost * days;
+
+  if (days < 3) {
+    return lessThanThreeDaysCost;
+  } 
+  
+  if (days < 7) {
+    return lessThanThreeDaysCost - discountThreeDays;
+  }
+
+  return lessThanThreeDaysCost - discountSevenDays;
```

Comment:
You're substractig `seven days dicount` from `less than three days cost`, other people would find it hard to understand what's going on here, make variable names more abstract